starport/cmd/network_request_show.go

```diff
@@ -0,0 +1,73 @@
+package starportcmd
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/yaml"
+)
+
+// NewNetworkRequestShow creates a new request show command to show
+// requests details for a chain
+func NewNetworkRequestShow() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "show [launch-id] [request-id]",
+		Short: "Show pending requests details",
+		RunE:  networkRequestShowHandler,
+		Args:  cobra.ExactArgs(2),
+	}
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	c.Flags().AddFlagSet(flagNetworkFrom())
+	c.Flags().AddFlagSet(flagSetHome())
+	return c
+}
+
+func networkRequestShowHandler(cmd *cobra.Command, args []string) error {
+	// initialize network common methods
+	nb, err := newNetworkBuilder(cmd)
+	if err != nil {
+		return err
+	}
+	defer nb.Cleanup()
+
+	// parse launch ID
+	launchID, err := strconv.ParseUint(args[0], 10, 64)
+	if err != nil {
+		return errors.Wrap(err, "error parsing launchID")
+	}
+	if launchID == 0 {
+		return errors.New("launch ID must be greater than 0")
+	}
+
+	// parse request ID
+	requestID, err := strconv.ParseUint(args[1], 10, 64)
+	if err != nil {
+		return errors.Wrap(err, "error parsing requestID")
+	}
+
+	n, err := nb.Network()
+	if err != nil {
+		return err
+	}
+
+	request, err := n.Request(cmd.Context(), launchID, requestID)
+	if err != nil {
+		return err
+	}
+
+	// convert the request object to YAML to be more readable
+	// and convert the byte array fields to string.
+	requestYaml, err := yaml.Marshall(cmd.Context(), request,
```

Comment:
Not a blocker but do you think it could be possible to remove the nested `content` in the object? I believe we got it because of the `oneof` usage on `spn`, it just looks weird

Reply:
Yes, I opened an issue for we can handle it later

Reply:
#1884